include/tvm/runtime/vm.h

```diff
@@ -405,6 +405,15 @@ class VirtualMachine : public runtime::ModuleNode {
   /*! \brief The set of TVM contexts the VM is currently executing on. */
   std::vector<TVMContext> ctxs;
 
+  /*! \brief The inputs to "main" function. */
```

Comment:
Can we combine these three members with `params_` somehow? `params_` is used to store both the parameters and inputs as well.